--- origsrc/glabels-2.2.2/libglabels/Makefile.am	2007-12-25 00:15:13.000000000 -0600
+++ src/glabels-2.2.2/libglabels/Makefile.am	2008-02-23 20:51:40.218750000 -0600
@@ -8,9 +8,10 @@
 
 libglabels_la_LDFLAGS=\
         -version-info $(LIBGLABELS_VERSION_INFO) \
-        $(LIBGLABELS_LIBS) \
         -no-undefined 
 
+libglabels_la_LIBADD = $(LIBGLABELS_LIBS)
+
 lib_LTLIBRARIES = libglabels.la
 
 libglabelsincludedir=$(includedir)/libglabels
--- origsrc/glabels-2.2.2/src/Makefile.am	2007-12-25 00:15:03.000000000 -0600
+++ src/glabels-2.2.2/src/Makefile.am	2008-02-23 20:51:40.218750000 -0600
@@ -25,9 +25,9 @@
 glabels_LDFLAGS = -export-dynamic
 
 glabels_LDADD = 				\
-	$(GLABELS_LIBS)				\
 	../libglabels/libglabels.la		\
 	mygal/libmygal.la			\
+	$(GLABELS_LIBS)				\
 	$(LIB_BARCODE_DIR)/libbarcode.la 	\
 	$(LIB_IEC16022_DIR)/libiec16022.la
 
