--- origsrc/glabels-2.2.2/src/Makefile.am	2007-12-25 00:15:03.000000000 -0600
+++ src/glabels-2.2.2/src/Makefile.am	2008-02-23 20:51:40.218750000 -0600
@@ -8,8 +8,8 @@
 INCLUDES = \
 	-I$(top_srcdir) 					\
 	-I$(top_builddir) 					\
-	-I$(LIB_BARCODE_DIR)					\
-	-I$(LIB_IEC16022_DIR)					\
+	-I$(srcdir)/$(LIB_BARCODE_DIR)					\
+	-I$(srcdir)/$(LIB_IEC16022_DIR)					\
 	$(GLABELS_CFLAGS) 					\
 	$(DISABLE_DEPRECATED_CFLAGS)				\
 	-DPREFIX=\""$(prefix)"\"                        	\
